rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/license/LicenseController.java

```diff
@@ -254,11 +254,15 @@ public ResponseEntity deleteAllLicense() throws TException {
             tags = {"Licenses"}
     )
     @PreAuthorize("hasAuthority('WRITE')")
-    @RequestMapping(value = LICENSES_URL + "/import/SPDX", method = RequestMethod.POST)
+    @RequestMapping(value = LICENSES_URL + "/import/SPDX", method = RequestMethod.GET)
```

Comment:
Why do we need to change from **POST** to **GET**. I think POST is more suitable.

Reply:
@hoangnt2 , I modified the code After discussion  to UI team, because state is not getting change here.

Reply:
@nikkuma7, return data after importing is fine, but method should be POST because it adds data to database